x/stakedym/types/dymension.go

```diff
@@ -0,0 +1,20 @@
+// #nosec G101
+package types
```

Comment:
verified everything but the addresses. Do you have a doc I can use for the addresses?